common/config/rush/common-versions.json

```diff
@@ -56,7 +55,7 @@
     // ]
     // Following is required to allow for backward compatibility with Event Processor Host Track 1
     "@azure/event-hubs": ["^2.1.4"],
-    "@opentelemetry/api": ["^0.14.0"],
+    "@opentelemetry/api": ["^0.16.0"],
```

Comment:
@richardpark-msft Let's make sure to remove this once #13573 is done. We may choose to simply comment it out as well in case the Azure Monitor package will be moving at a higher speed than the rest of the packages when it comes to this dependency